src/main/java/com/example/onlinebookstore/model/Book.java

```diff
@@ -9,10 +9,14 @@
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Getter
 @Setter
+@SQLDelete(sql = "UPDATE book SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted = false")
```

Comment:
`@Where` is deprecated, let's use `@SqlRestriction` instead, and if it won't work try changing the `is_deleted` type to `bit` in liquibase